rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java

```diff
@@ -2564,9 +2551,28 @@ public BigInteger registerFastBridgeBtcTransaction(
         Address fastBridgeFedAddress =
             fastBridgeFederationInformation.getFastBridgeFederationAddress(bridgeConstants.getBtcParams());
 
-        Coin totalAmount = getAmountSentToAddress(btcTx, fastBridgeFedAddress);
-
-        if (totalAmount == Coin.ZERO) {
+        Coin totalAmount;
+        Federation retiringFederation = getRetiringFederation();
+        if (!activations.isActive(ConsensusRule.RSKIP293) || retiringFederation == null) {
```

Comment:
```suggestion
        if (activations.isActive(ConsensusRule.RSKIP293) && retiringFederation != null) {
```
Just a suggestion, maybe it's just me but I find it clearer when the `if` statement has the 'positive'  condition